06_puppy/hsy3418/store_test.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
```

Comment:
I'm curious why did you defined methods to create puppies as below. You could have stored them in variables in `suite` and created at `SetupTest()`.